app/src/main/java/mozilla/lockbox/action/FingerprintAuthAction.kt

```diff
@@ -7,9 +7,9 @@
 package mozilla.lockbox.action
 
 import mozilla.lockbox.flux.Action
-import mozilla.lockbox.model.FingerprintAuthCallback
 
 sealed class FingerprintAuthAction : Action {
-    data class OnAuthentication(val authCallback: FingerprintAuthCallback) : FingerprintAuthAction()
+    object OnAuthSuccess : FingerprintAuthAction()
```

Comment:
Rather than duplicating `Auth` in both the `FingerprintAuthAction` and `OnAuthSuccess` / `OnAuthError`, let's name these `OnSuccess` and `OnError`!